src/main/java/com/matt/forgehax/mods/AutoReply.java

```diff
@@ -47,7 +49,7 @@ public AutoReply() {
   @SubscribeEvent
   public void onClientChat(ClientChatReceivedEvent event) {
     String message = (event.getMessage().getUnformattedText());
-    if (message.contains(search.get()) && !message.startsWith(MC.getSession().getUsername())) {
+    if (Pattern.matches(search.get(), message)) && !message.startsWith(MC.getSession().getUsername())) {
```

Comment:
Your build is failing here because you have an extra `)` at `...message))`. You could have just replaced `message.contains` with `message.matches` btw.